src/org/zaproxy/zap/extension/websocket/ui/httppanel/models/StringWebSocketPanelViewModel.java

```diff
@@ -28,24 +28,29 @@
 	
 	private static final Logger LOGGER = Logger.getLogger(StringWebSocketPanelViewModel.class);
 	private boolean isErrorMessage;
-
+	private boolean isEditable = true;
+	
     @Override
     public String getData() {
     	String data;
         if (webSocketMessage == null || webSocketMessage.payload == null)  {
             data = "";
         } else {

```

Comment:
`else if (!editable) {` instead.

Reply:
Bump, I think using an `else if` is better, it avoids nesting the other statements even more.

Reply:
Sorry I should be more careful

Reply:
No problem.